build.gradle

```diff
@@ -29,6 +29,7 @@ project.ext.dependencyStrings = [
   MAVEN_EXTENSION: 'com.google.cloud.tools:jib-maven-plugin-extension-api:0.4.0',
 
   COMMONS_COMPRESS: 'org.apache.commons:commons-compress:1.21',
+  ZSTD_JNI: 'com.github.luben:zstd-jni:1.5.2-3',
```

Comment:
A third-party dependency does not seem ideal. Is there a recommended oci library?